Stufe 5/labor/feaure/ptdata 1938 natrium

alexzautke opened the pull request:
Erster Prototyp eines Profils für Serumnatrium, das als Parameter im Angestrebten Modul "Unterstützung Transplantationsbeauftrage" als OPTIONAL gekennzeichnet ist.

Solves https://service.gematik.de/browse/PTDATA-1938

alexzautke committed:
Add profile “ISiKLaboruntersuchungSerumnatrium”

alexzautke committed:
Add SUHSI output

alexzautke committed:
Merge branch 'main-stufe-5' into stufe-5/labor/feaure/PTDATA-1938-natrium

alexzautke committed:
ISiKLaboruntersuchungSerumnatrium -> Optional

alexzautke committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

alexzautke committed:
Add optional profile for serum sodium mapping

f-peverali commented on Resources/input/fsh/Labor/ISiKLaboruntersuchungSerumnatrium.fsh:
```diff
@@ -0,0 +1,28 @@
+Profile: ISiKLaboruntersuchungSerumnatrium
+Parent: ISiKLaboruntersuchung
+Id: ISiKLaboruntersuchungSerumnatrium
+Description: "Dieses Profil ermöglicht die Abbildung der Laboruntersuchung Serumnatrium eines Patienten in ISiK Szenarien."
```
comment: hier sollte u. A. der UC zur Unterstützung der Transplantationsbeauftragten zur Spendererkennung genannt werden.

Zudem soll hier dokumentiert werden, dass dieser Wert nicht notwendigerweise aus einem Labor kommt, sondern z. B. unmittelbar auf Intensivstation mittels HW ermittelt werden und dem PDMS übermittelt werden kann.

alexzautke replied:
Done

f-peverali commented on Resources/input/fsh/Labor/ISiK-CapabilityStatementLaborMinimalRolle.fsh:
```diff
@@ -41,6 +41,8 @@ Usage: #definition
       * insert CapabilityStatementExpectationExt(SHALL)
     * supportedProfile[+] = Canonical(ISiKLaboruntersuchungTSH)
       * insert CapabilityStatementExpectationExt(SHALL)
+    * supportedProfile[+] = Canonical(ISiKLaboruntersuchungSerumnatrium)
```
praise

f-peverali replied:
... an sich gut

f-peverali commented on Resources/input/fsh/Labor/ISiK-CapabilityStatementLaborMinimalRolle.fsh:
```diff
@@ -41,6 +41,8 @@ Usage: #definition
       * insert CapabilityStatementExpectationExt(SHALL)
     * supportedProfile[+] = Canonical(ISiKLaboruntersuchungTSH)
       * insert CapabilityStatementExpectationExt(SHALL)
+    * supportedProfile[+] = Canonical(ISiKLaboruntersuchungSerumnatrium)
```
question: für das Modul "Unterstützung Transplantationsbeaftragte" mittels Parameter-Auswertung sollte auch "MAY" genügen, da als "optional" gekennzeichnet? Ggf. als SHOULD (SOLLTE statt SHALL (MUSS)?

alexzautke replied:
Done

alexzautke committed:
Update description for ISiKLaboruntersuchungSerumnatrium

Expanded description to include use case for transplant support.

alexzautke committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

alexzautke committed:
Revise description for ISiKLaboruntersuchungSerumnatrium

Updated the description to clarify the use case and context for Point-of-Care-Testing.

alexzautke committed:
Merge branch 'main-stufe-5' into stufe-5/labor/feaure/PTDATA-1938-natrium

alexzautke committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

alexzautke committed:
Change CapabilityStatementExpectationExt from MAY to SHOULD

alexzautke committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

f-peverali commented on Resources/input/fsh/Labor/ISiKLaboruntersuchungSerumnatrium.fsh:
```diff
@@ -0,0 +1,28 @@
+Profile: ISiKLaboruntersuchungSerumnatrium
+Parent: ISiKLaboruntersuchung
+Id: ISiKLaboruntersuchungSerumnatrium
+Description: "Dieses Profil ermöglicht die Abbildung der Laboruntersuchung Serumnatrium eines Patienten in ISiK Szenarien. Das Profil wird unter anderem im Use Case zur Unterstützung von Transplantationsbeauftragten durch DETECT (siehe https://www.detect-iha.de) eingesetzt, insbesondere im Rahmen von Messungen mittels Point-of-Care-Testing."
```
```suggestion
Description: "Dieses Profil ermöglicht die Abbildung der Laboruntersuchung Serumnatrium eines Patienten in ISiK Szenarien. Das Profil wird unter anderem im Use Case zur Unterstützung von Transplantationsbeauftragten bei der Organspendeerkennung eingesetzt; besonders in diesem Kontext MUSS es auch Werte abbilden, die im Rahmen von Messungen mittels Point-of-Care-Testing erhoben wurden."
```

f-peverali committed:
Apply suggestion from @f-peverali

f-peverali commented on Resources/input/fsh/Labor/ISiKLaboruntersuchungSerumnatrium.fsh:
```diff
@@ -0,0 +1,28 @@
+Profile: ISiKLaboruntersuchungSerumnatrium
+Parent: ISiKLaboruntersuchung
+Id: ISiKLaboruntersuchungSerumnatrium
+Description: "Dieses Profil ermöglicht die Abbildung der Laboruntersuchung Serumnatrium eines Patienten in ISiK Szenarien. Das Profil wird unter anderem im Use Case zur Unterstützung von Transplantationsbeauftragten bei der Organspendeerkennung eingesetzt; besonders in diesem Kontext MUSS es auch Werte abbilden, die im Rahmen von Messungen mittels Point-of-Care-Testing erhoben wurden."
```
```suggestion
Description: "Dieses Profil ermöglicht die Abbildung der Laboruntersuchung Serumnatrium eines Patienten in ISiK Szenarien. Das Profil wird u. A. im Use Case zur Unterstützung von Transplantationsbeauftragten bei der Organspendeerkennung eingesetzt; besonders in diesem Kontext muss es auch Werte abbilden, die im Rahmen von Messungen mittels Point-of-Care-Testing erhoben wurden. Das Profil ist auch geeignet, um Serumnatrium Werte abzubilden, die mittels Laboruntersuchung erhoben wurden"
```

f-peverali committed:
Apply suggestion from @f-peverali

f-peverali commented on Resources/input/fsh/Labor/ISiKLaboruntersuchungSerumnatrium.fsh:
```diff
@@ -0,0 +1,28 @@
+Profile: ISiKLaboruntersuchungSerumnatrium
+Parent: ISiKLaboruntersuchung
+Id: ISiKLaboruntersuchungSerumnatrium
+Description: "Dieses Profil ermöglicht die Abbildung der Laboruntersuchung Serumnatrium eines Patienten in ISiK Szenarien. Das Profil wird u. A. im Use Case zur Unterstützung von Transplantationsbeauftragten bei der Organspendeerkennung eingesetzt; besonders in diesem Kontext muss es auch Werte abbilden, die im Rahmen von Messungen mittels Point-of-Care-Testing erhoben wurden. Das Profil ist auch ...
```
question: da hier eine Differenzierung bei der Erhebung vorgesehen ist; ggf. sinnvoll diese auch über ein Element im Profil abzubilden (POCT vs Labor)?
Bzw. was sind ggf. spezifische Eigenarten, für unterschiedlich erhobene Werte?

alexzautke replied:
Mir würde außer der einer unterschiedlichen Kodierung von .method nichts einfallen. @patrick-werner Fällt dir etwas ein?

opfps replied:
Ich erinnere mich an eine Diskussion, in der ein PDMS eine BGA unterschiedlich zu einer Laboruntersuchung handhabte. Bei der Laboruntersuchung müsste ein Arzt die Werte zuerst sichten und bestätigen, eh sie im PDMS als "final" für den Patienten hinterlegt werden.

f-peverali committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

f-peverali committed:
Merge branch 'main-stufe-5' into stufe-5/labor/feaure/PTDATA-1938-natrium

f-peverali committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

f-peverali committed:
Merge branch 'main-stufe-5' into stufe-5/labor/feaure/PTDATA-1938-natrium

f-peverali committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

f-peverali commented on Resources/input/fsh/Labor/ISiKLaboruntersuchungSerumnatrium.fsh:
```diff
@@ -0,0 +1,28 @@
+Profile: ISiKLaboruntersuchungSerumnatrium
+Parent: ISiKLaboruntersuchung
+Id: ISiKLaboruntersuchungSerumnatrium
+Description: "Dieses Profil ermöglicht die Abbildung der Laboruntersuchung Serumnatrium eines Patienten in ISiK Szenarien. Das Profil wird u. A. im Use Case zur Unterstützung von Transplantationsbeauftragten bei der Organspendeerkennung eingesetzt; besonders in diesem Kontext muss es auch Werte abbilden, die im Rahmen von Messungen mittels Point-of-Care-Testing erhoben wurden. Das Profil ist auch ...
```
```suggestion
Description: "Dieses Profil ermöglicht die Abbildung der Laboruntersuchung Serumnatrium eines Patienten in ISiK Szenarien. Das Profil wird u. A. im Use Case zur Unterstützung von Transplantationsbeauftragten bei der Organspendeerkennung eingesetzt; besonders in diesem Kontext muss es auch Werte abbilden, die im Rahmen von Messungen mittels Point-of-Care-Testing erhoben wurden. Das Profil ist auch geeignet, um Serumnatrium Werte abzubilden, die mittels Laboruntersuchung erhoben wurden.

Eine eindeutige Kennzeichnung für die Differenzierung hinsichtlich der Erhebungsmethode ist derzeit über dieses Profil nicht vorgesehen. Es kann jedoch das Element .method verwendet werden. Dies kann unter Umständen sinnvoll sein, wenn im Falle einer Laboruntersuchung ein Arzt die Werte zuerst sichten und bestätigen müsste, bevor sie im PDMS als "final" für den Patienten hinterlegt werden. "
```

f-peverali committed:
Apply suggestion from @f-peverali
